src/main/resources/db/migration/R__Comments.sql

```diff
@@ -465,8 +465,11 @@ COMMENT ON TABLE accelerator.participants IS 'Accelerator participant details.';
 COMMENT ON TABLE accelerator.project_document_settings IS 'Per-project configuration for storage of submitted documents.';
 COMMENT ON COLUMN accelerator.project_document_settings.file_naming IS 'Identifier that is included in generated filenames. This is often, but not necessarily, the same as the project name.';
 
+COMMENT ON TABLE accelerator.project_scores IS 'Score options (nullable) selected by voters.';
```

Comment:
Blank line between these two just so each table's comment(s) are visually separated from other tables.

Reply:
Also, is the "(nullable)" accurate?

Reply:
Nullable is accurate because a scorer can select a 'Not Assigned Score Yet' state essentially

Reply:
I separated the comments to one about the score column itself. Should give a bit of clarity